xslt/uddf.xslt

```diff
@@ -174,15 +174,15 @@
         <xsl:if test="//u:divesite/u:site[@id = $ref]/u:name">
           <location>
             <xsl:choose>
-            <xsl:when test="//u:divesite/u:site[@id=$ref]/u:geography/u:longitude != ''">
-              <xsl:attribute name="gps">
-                <xsl:value-of select="concat(//u:divesite/u:site[@id=$ref]/u:geography/u:latitude, ' ', //u:divesite/u:site[@id=$ref]/u:geography/u:longitude)"/>
-              </xsl:attribute>
-              <xsl:value-of select="//u:divesite/u:site[@id=$ref]/u:name"/>
-            </xsl:when>
-            <xsl:otherwise>
-              <xsl:value-of select="//u:divesite/u:site[@id=$ref]/u:name"/>
-            </xsl:otherwise>
+              <xsl:when test="//u:divesite/u:site[@id=$ref]/u:geography/u:longitude != ''">
+                <xsl:attribute name="gps">
+                  <xsl:value-of select="concat(//u:divesite/u:site[@id=$ref]/u:geography/u:latitude, ' ', //u:divesite/u:site[@id=$ref]/u:geography/u:longitude)"/>
+                </xsl:attribute>
+                <xsl:value-of select="//u:divesite/u:site[@id=$ref]/u:name"/>
+              </xsl:when>
+              <xsl:otherwise>
+                <xsl:value-of select="//u:divesite/u:site[@id=$ref]/u:name"/>
+              </xsl:otherwise>
             </xsl:choose>
```

Comment:
You should get an award for doing this. Thanks - so much easier to read and understand this way.